addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -310,7 +314,6 @@ patch(Chatter.prototype, {
     onFollowerChanged(thread) {
         document.body.click(); // hack to close dropdown
         this.reloadParentView();
-        this.load(thread, ["followers", "suggestedRecipients"]);
```

Comment:
Removing it from here will prevent proper reload in `onClickEdit` and `onClickApply` from dialog if we don't also update the list in these respective flows.
(It looks like we have a bad test coverage.)

Reply:
It look like it call /mail/data asking for everything:
![image](https://github.com/user-attachments/assets/727f4306-ed95-49d6-968c-0845f041b091)